### PR TITLE
feat: dvorak support

### PR DESCRIPTION
**Pull request by tariks**
Untested Dvorak support.

Note: Dvorak has punctuation at some useful keymap positions.
Note: Dvorak best layout.
Note: Viva la Dvoración.

### CODE REVIEW

**Comment by tris203** [comment]
Thanks so much for your contribution @tariks 

I will try and review this. Did you want to add your name to the keyboard contributors in README?

**Review by willothy — APPROVED** [review]
I'm not a dvorak user so can't verify the Dvorak layout, but looks good to me!

**Comment by tris203** [comment]
Looks good to me too. Just the documentation changes in README to show support and add @tariks to the contributors if he wants to be there

**Comment by tariks** [comment]
Sure, add me to the readme. Thank you :)

No issues encountered during personal use. Scores make sense for the layout. Great plugin

**Comment by tris203** [comment]
You can do it yourself @tariks

The README in In the repo. If not I'll do it shortly

File: lua/hawtkeys/keyboards/dvorak.lua [hunk 1]
```diff
@@ -0,0 +1,50 @@
+
```
Comment (tris203) [inline lua/hawtkeys/keyboards/dvorak.lua]:
```suggestion
```

remove this blank line for the stylua checks to pass

**Review by tris203 — CHANGES_REQUESTED** [review]
Thanks for your contribution!

**Comment by tris203** [comment]
Thanks so much!

**Pull request merged**